### DISCUSSION TITLE
Share your WinBox machine templates here!

### DISCUSSION THREAD

**Discussion by swordlink1** [86Box/86Box#2166] [General]
Basically, WinBox machine templates are pre-defined machines that you can choose while making a new machine (and customize it specs in the new machine configuration window if supported). Like if you choose the Compaq Deskpro template, and chose 640kb of ram, you would get a new VM emulating a Compaq Deskpro, with 640kb of RAM supplied.

**Comment by waltje** [comment]
How is this relevent to 86Box ?
Besides.. this smells a lot like MAME with its pre-defined (or even authorized,I cannot remember) packages.

**Reply by swordlink1** [reply]
I've never used MAME, but I'm just going to believe you.
Plus, give a link to the WinBox discussion page, maybe then I can make a more relevant post.

**Comment by waltje** [comment]
I have no idea where WinBox lives, just that it's not 86Box.
--waltje

 
I've never used MAME, but I'm just going to believe you.
Plus, give a link to the WinBox discussion page, maybe then I can make a more relevant post.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by jriwanek** [comment]
https://github.com/86Box/WinBox-for-86Box

**Comment by leuamS** [comment]
i don't see a discussion tab on the winbox page

**Reply by ts-korhonen** [reply]
Don't worry about the off-topic comments; WinBox for 86Box is very much relevant to 86Box and can be discussed here.

**Comment by swordlink1** [comment]
So is anyone gonna share their templates, or refuse to because I didn't post it in the WinBox discussion page? (because i can't)

**Comment by leuamS** [comment]
ASUS P/I-P54TP4XE
32MB
WinChip 75 MHZ
S3 Trio
AudioPCI
That's the slowest I've seen XP run

**Reply by swordlink1** [reply]
Okay, by that, I meant those templates in Winbox to select a machine, but it counts I guess.
